### PR TITLE
Guardian AI

### PR DESCRIPTION
**Pull request by TheErrrors**

### CODE REVIEW

File: ai.html [hunk 1]
```diff
@@ -0,0 +1,600 @@
...
... 12.447 23 13 23 z M 37.1875 30.019531C37.315281 30.043891 37.441141 30.093172 37.556641 30.169922C38.015641 30.474922 38.139031 31.094688 37.832031 31.554688L33.832031 37.554688C33.802031 37.599688 33.765469 37.635828 33.730469 37.673828C33.720469 37.683828 33.717031 37.697031 33.707031 37.707031C33.702031 37.712031 33.696406 37.71375 33.691406 37.71875C33.606406 37.80075 33.507344 37.862203 33.402344 37.908203C33.379344 37.918203 33.357984 37.927547 33.333984 37.935547C33.226984 37.973547 33.115953 38 33.001953 38L33 38L32.998047 38C32.885047 38 32.773062 37.974547 32.664062 37.935547C32.630063 37.923547 32.600359 37.906625 32.568359 37.890625C32.527359 37.870625 32.484312 37.857031 32.445312 37.832031C32.416312 37.813031 32.402953 37.784672 32.376953 37.763672C32.350953 37.742672 32.318922 37.732031 32.294922 37.707031L29.294922 34.707031C28.903922 34.316031 28.903922 33.685922 29.294922 33.294922C29.685922 32.903922 30.316031 32.903922 30.707031 33.294922L32.845703 35.431641L36.169922 30.447266C36.399422 30.102266 36.804156 29.946453 37.1875 30.019531 z M 13 33L24 33C24.553 33 25 33.447 25 34C25 34.553 24.553 35 24 35L13 35C12.447 35 12 34.553 12 34C12 33.447 12.447 33 13 33 z"
```
Comment (ivelin) [inline ai.html:211]: This does not look great. Mixing binary data into source code.
Hard to maintain embedded binary data. Why not link to a local or CDN svg icon file with a recognizable name?

**Review by ivelin — CHANGES_REQUESTED** [review]
Looks good except for embedded svg data. Please use friendly named links to svg icons via local files or CDN urls.

**Comment by lipmaneth** [comment]
recording

Also the "Model Training" section header and copy should be above the second image for mobile view. It is currently underneath the image. 
<details><summary>Browser metadata</summary>





```
Path:      /ai.html
Browser:   Chrome 109.0.0.0 on Mac OS 10.15.7
Viewport:  1166 x 978 @2x
Language:  en-US
Cookies:   Enabled
```

[Open in BrowserStack](https://www.browserstack.com/user/try-live?url=https%3A%2F%2Fdeploy-preview-25--lustrous-snickerdoodle-f0ebfa.netlify.app%2Fai.html&os=OS%20X&os_version=Catalina&browser=Chrome&browser_version=109.0&resolution=1166x978&speed=1&start=true&ref=netlify-source)



</details>

[Open Deploy Preview](https://deploy-preview-25--lustrous-snickerdoodle-f0ebfa.netlify.app/ai.html) · [Mark as Resolved](https://app.netlify.com/cdp/resolve?deployID=63f9bed0d32ed3000852f6fa&commentID=63fae1e69a249e2322af93b2&resolution=resolved)

**Comment by ivelin** [comment]
@lipmaneth let me know when you review this. I'll go next.

**Pull request merged**